### ISSUE TITLE
Add support for custom headers

### ISSUE THREAD

**Issue by fabasoad** [fjogeleit/http-request-action#1]
Would be great to have a support for custom headers.

**Comment by fjogeleit** [comment]
I added a new property "customHeaders" to add additional headers as JSON string. Its available in master or the new version v1.1.0

See the documentation for detailed usage

**Comment by harmanpa** [comment]
Using v1.3.2 with a ```customHeaders``` input I'm getting the following error:
```
##[warning]Unexpected input(s) 'customHeaders', valid inputs are ['url', 'method', 'contentType', 'data', 'username', 'password', 'timeout', 'bearerToken']
```
